Spigot-Server-Patches/0595-added-PlayerChunkReceiveEvent-logic.patch

```diff
@@ -0,0 +1,31 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: ysl3000 <yannicklamprecht@live.de>
+Date: Mon, 5 Oct 2020 21:25:16 +0200
+Subject: [PATCH] added PlayerChunkReceiveEvent logic
```

Comment:
All of these impl patches shouldnt say "added", 

`"<EventName>"` by itself is fine or

"Player Chunk Load/Unload Events"
"Player Entity Tracking Events"

Reply:
Applied.